var/spack/repos/builtin/packages/hdf5/package.py

```diff
@@ -97,6 +98,9 @@ class Hdf5(AutotoolsPackage):
     patch('h5f90global-mult-obj-same-equivalence-same-common-block.patch',
           when='@1.10.1%intel@18')
 
+    patch('th5s.patch', when='@1.8.10')
```

Comment:
Here it would be nice if you could add a few words that explain the purpose of the patch, like:
```
# Turn line comments into block comments that are conformant with ISO C 89
```